simulator/func_sim/syscall/mars_calls.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+ * mars_calls.cpp - implementation of MARS syscalls
+ * @author Vyacheslav Kompan kompan.vo@phystech.edu
+ * Copyright 2018 MIPT-MIPS
+ */
+
+#include "mars_calls.h"
+/* ISA */
+#include <mips/mips.h>
+/* Generic C & C++ */
+#include <cassert>
+#include <utility>
+
+static const MIPSRegister v0 = MIPSRegister::from_cpu_index( 2);
+static const MIPSRegister a0 = MIPSRegister::from_cpu_index( 4);
+
+std::pair<bool, long long> strToInt( const std::string &str) {
+    char *end;
+    long long value = std::strtoll( str.c_str(), &end, 0);
```

Comment:
You are looking for `stoll` here

https://en.cppreference.com/w/cpp/string/basic_string/stol

Reply:
Thanks, I didn't know about it.